AppCenterDataStorage/AppCenterDataStorage/Internal/Ingestion/MSCosmosDbIngestion.m

```diff
@@ -0,0 +1,52 @@
+#import "MSCosmosDbIngestion.h"
+#import "MSAppCenter.h"
+#import "MSAppCenterInternal.h"
+#import "MSHttpIngestionPrivate.h"
+#import "MSLoggerInternal.h"
+
+@implementation MSCosmosDbIngestion
+
+/**
+ * Http header Authorization value.
+ */
+static NSString *const kMSAppSecrectHeader = @"Authorization";
+
+- (instancetype)init {
+  if ((self = [super initWithBaseUrl:@""
+                             apiPath:@""
+                             headers:@{}
+                        queryStrings:nil
+                        reachability:[MS_Reachability reachabilityForInternetConnection]
+                      retryIntervals:@[ @(10), @(5 * 60), @(20 * 60) ]])) {
```

Comment:
Cosmos DB gives us a header back called "x-ms-retry-after-ms" and a status code of 429. We need to retry with a delay based on what we get back from cosmos.

Reply:
good point, I'll track this as WI:
https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/58169